tailwind.config.js

```diff
@@ -335,6 +337,7 @@ module.exports = {
         h4: '1.125rem',
         h5: '1rem',
         h6: '0.875rem',
+        '6px': '0.375rem',
```

Comment:
I'm asking designer to fix font size

Reply:
I think we can use that font size for now and then when our designer has been updated we can get back and update it.

Reply:
Had a discussion and concluded exactly same as you thought. Approving this now.